api-extractor/src/definitions/ApiItem.ts

```diff
@@ -6,6 +6,52 @@ import Extractor from '../Extractor';
 import ApiDocumentation from './ApiDocumentation';
 
 /**
+ * Indicates the type of definition represented by a ApiItem object.
+ */
+export enum ApiItemKind {
+  /**
+    * A TypeScript class.
+    */
+  class = 0,
+  /**
+    * A TypeScript enum.
```

Comment:
In your comment, it might also be useful to say which ApiItem class goes with each enum.  E.g. Interface, Class, TypeLiteral are all ApiStructuredType